react-native-scripts/README.md

```diff
@@ -7,3 +7,21 @@ The README for this package is hosted at https://github.com/react-community/crea
 `yarn && yarn start` will start a watcher that will build artifacts and place them in the build directory.
 
 *Warning*: if you change the template project's dependencies, make sure to update `checkAppName` in `create-react-native-app/src/index.js`.
+
+## Adding Flow
```

Comment:
Sorry, should have specified before, this needs to go in the template README for users to get it in their apps https://github.com/react-community/create-react-native-app/blob/master/react-native-scripts/template/README.md